### ISSUE TITLE
Magda, new Polish voice.

### ISSUE THREAD

**Issue by grzezlo** [RHVoice/RHVoice#444]
We prepared a voice for Polish language, based on recording of good-voiced female speaker in the professional studio.
All authors including the speaker, agreed to make it available for RHVoice, licensed under Creative Commons, no derivatives, non-commercial.

What should I do next?

Should I create the voice repo on my Github account and put the files there, creating a pull-request adding the voice as submodule?

Voice sample:
https://zlotowicz.pl/files/SyntezaMagdaSample.mp3

**Comment by beqabeqa473** [comment]
Hello.

Based on experience of creating voices and sound processing, which
most of the time could drasticly improve voice qualite, on behave of
RHVoice lab, we are glad to offer you collaboration.

If you agree, we can try to further improve voice quality based on our
experience, with help of our sound engineers and avoid some unpleasant
artefacts.

Thanks.

On 1/18/22, grzezlo ***@***.***> wrote:
> We prepared a voice for Polish language, based on recording of good-voiced
> female speaker in the professional studio.
> All authors including the speaker, agreed to make it available for RHVoice,
> licensed under Creative Commons, no derivatives, non-commercial.
>
> What should I do next?
>
> Should I create the voice repo on my Github account and put the files there,
> creating a pull-request adding the voice as submodule?
>
> Voice sample:
> https://zlotowicz.pl/files/SyntezaMagdaSample.mp3
>
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/RHVoice/RHVoice/issues/444
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Comment by alex19EP** [comment]
> Should I create the voice repo on my Github account

i will create repo in rhvoice org for you.

**Comment by grzezlo** [comment]
@beqabeqa473 , thanks for your offer.
As I mentioned elsewhere, you helped us a lot at the beginning of RHVoice adventure and I'm very grateful for it.

The audio processing is done by my coleague, Roman. He would lose a lot of fun if he would give it to be done by others.

The voice sample I linked is few days old, since there we have added an equalizer improving the audio.
We've some other ideas of improving the quality, including one more studio session, to have more voice-training material, but of course any hints and suggestions regarding audio processing and training improvements, are very welcome.

@alex19EP, thanks for creating the repo and inviting me. I'll upload the voice data soon.
